nrf_security/src/mbedtls/vanilla/CMakeLists.txt

```diff
@@ -4,6 +4,10 @@
 # SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
 #
 
+if(TARGET platform_cc310)
+  get_target_property(NRF_PLATFORM_CC310_INCLUDE_PATH platform_cc310 BASE_INCLUDE_PATH)
+endif()
```

Comment:
This is already safe guarded at line 69:
https://github.com/nrfconnect/sdk-nrfxlib/blob/cf2ee6faf59402fce24d4c2f75e78e1849bf81cf/nrf_security/src/mbedtls/shared/CMakeLists.txt#L69

So why not simply use a generator expression and remove those 3 lines of code ?
```
if(TARGET platform_cc310)
  target_include_directories(mbedcrypto_shared PRIVATE
    ${mbedcrypto_glue_include_path}/threading
    $<TARGET_PROPERTY:platform_cc310,BASE_INCLUDE_PATH>
  )
endif()
```

Reply:
Great. Will change to that!

Reply:
Fixed